Resources/views/admin/partials/permissions.blade.php

```diff
@@ -7,59 +7,39 @@
-                            <p class="pull-right" style="margin-top: 10px;">
-                                <a href="" class="jsSelectAllInGroup">{{ trans('user::roles.select all')}}</a> |
-                                <a href="" class="jsDeselectAllInGroup">{{ trans('user::roles.deselect all')}}</a> |
-                                <a href="" class="jsSwapAllInGroup">{{ trans('user::roles.swap')}}</a>
-                            </p>
-                        </div>
-                        <div class="clearfix"></div>
-                        <?php foreach (array_chunk($permissionActions, ceil(count($permissionActions)/2)) as $permissionActionGroup): ?>
-                            <div class="col-md-3">
-                                <?php foreach ($permissionActionGroup as $permissionAction): ?>
-                                    <div class="checkbox">
-                                        <label for="<?php echo "$subPermissionTitle.$permissionAction" ?>">
-                                            <input name="permissions[<?php echo "$subPermissionTitle.$permissionAction" ?>]" type="hidden" value="false" />
-                                            <input id="<?php echo "$subPermissionTitle.$permissionAction" ?>" name="permissions[<?php echo "$subPermissionTitle.$permissionAction" ?>]" type="checkbox" class="flat-blue" <?php echo array_get($model->permissions, "$subPermissionTitle.$permissionAction", false) === true ? 'checked' : '' ?> value="true" /> {{ ucfirst($permissionAction) }}
-                                        </label>
-                                    </div>
-                                <?php endforeach; ?>
-                            </div>
-                        <?php endforeach; ?>
-                        <div class="clearfix"></div>
+                <?php foreach (array_chunk($permissionActions, ceil(count($permissionActions)/2)) as $permissionActionGroup): ?>
+                <div class="col-md-3">
+                    <?php foreach ($permissionActionGroup as $permissionAction): ?>
+                    <div class="checkbox">
+                        <label for="<?php echo "$subPermissionTitle.$permissionAction" ?>">
+                            <input name="permissions[<?php echo "$subPermissionTitle.$permissionAction" ?>]" type="hidden" value="false" />
+                            <input id="<?php echo "$subPermissionTitle.$permissionAction" ?>" name="permissions[<?php echo "$subPermissionTitle.$permissionAction" ?>]" type="checkbox" class="flat-blue" <?php echo array_get($model->permissions, "$subPermissionTitle.$permissionAction", false) === true ? 'checked' : '' ?> value="true" /> {{ ucfirst($permissionAction) }}
+                        </label>
                     </div>
+                    <?php endforeach; ?>
+                </div>
                 <?php endforeach; ?>
+                <div class="clearfix"></div>
+            </div>
+            <?php endforeach; ?>
```

Comment:
I'm not sure what's up with this diff. The only changes I made are the same as on `permissions-create.blade.php`.